src/Storefront/Theme/ThemeCompiler.php

```diff
@@ -214,8 +213,7 @@ private function compileStyles(
             );
         }
         $autoPreFixer = new Autoprefixer($cssOutput);
-
-        return $autoPreFixer->compile();
+        return $autoPreFixer->compile(false);
```

Comment:
This causes minification also on debug mode, doesn't it? You could respect the `$debug` or scssCompiler::Formatter 🤔